sdk/database/dbplugin/grpc_transport.go

```diff
@@ -283,3 +308,57 @@ func (c *gRPCClient) Close() error {
 	_, err := c.client.Close(c.doneCtx, &Empty{})
 	return err
 }
+
+func (c *gRPCClient) GenerateCredentials(ctx context.Context) (string, error) {
+	ctx, cancel := context.WithCancel(ctx)
+	quitCh := pluginutil.CtxCancelIfCanceled(cancel, c.doneCtx)
+	defer close(quitCh)
+	defer cancel()
+
+	// TODO: Question: CreateUser takes the CreateUserRequest and splits the
+	// statements and the config. Here we just pass on the request without
+	// splitting anything
+	resp, err := c.client.GenerateCredentials(ctx, &Empty{})
+
+	if err != nil {
+		// Fall back to old call if not implemented
```

Comment:
Comment is out of date

Reply:
Removed in 2ca4849e3 it was bad copy-paste